src/NUnitFramework/framework/Internal/Results/TestResult.cs

```diff
@@ -268,7 +283,7 @@ internal set
                 InternalAssertCount = value;
             }
         }
-
+        
```

Comment:
I really hate to nitpick whitespace. I like these tools:
VS: https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer
VS Code: https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces with `"trailing-spaces.trimOnSave": true`